### DISCUSSION TITLE
cannot access files tracked by git-lfs

### DISCUSSION THREAD

**Discussion by Tomekmularczyk** [vercel/next.js#58352] [Help]
### Summary

Hi, I'm looking for help. I have a NEXTjs project hosted on Vercel using a free plan. 

I had to use git-lfs to push some heavy JSON files to a GitHub repository. I have git-lfs option turned on in project settings but 
when I try to use the hosted app I get this error in the logs: `Runtime.UnhandledPromiseRejection: Error: ENOENT: no such file or directory`.

This error does not happen if I build and start the app locally...

**Comment by zenpi-me** [comment]
I have the same problem with a LFS file in my project. The error message looks a bit different. (`Uncaught Exception: Error: spawn /vercel/path0/src/app/api/go/katago-v1.15.3-eigen-linux-x64/katago ENOENT`

Did you find a solution?
